CHANGELOG.md

```diff
@@ -678,6 +768,8 @@ Same changes as in `@mui/x-charts@7.12.1`, plus:
 
 #### `@mui/x-tree-view@7.12.1`
 
+No changes since `@mui/x-tree-view@7.12.0`.
```

Comment:
Would a hyperlink to the mentioned version make sense here?

Reply:
Not having a strong opinion on this one.
If you are scrolling through all releases, then it is the next one.
For the announcements, you probably already had the previous one so you don't need to remind yourself what was there.

Waiting for 👍 from the reviewers
>= 3 - I am adding it 😆

Reply:
For a changelog, yes it doesn't make sense, but when on a particular release page, it might be helpful to have a link if another release is mentioned. Let's not block the release for this though, we can add it later too. 👍